frontend/components/pod_components/AssociatePod.js

```diff
@@ -13,7 +13,14 @@ export default function AssociatePodScreen({ navigation }) {
             
             <TouchableOpacity 
                 style={styles.block} 
-                onPress={() => navigation.navigate('Random Screen')}
+                onPress={() => {
+                    const { pod } = props.route.params;
+                    props.navigation.navigate('Outcomes', {
+                        pod: pod,
+                        focus_area: "COM",
+                        title: "Competencies"
+                    });
+                }}
```

Comment:
These look good for now to get basic functionality! Nice we can finally start to see outcomes for different pods